### PR TITLE
Feature/splash login signup

### PR DESCRIPTION
**Pull request by SeaUrc**
Screen Shot 2022-07-10 at 7 56 18 PM
Screen Shot 2022-07-10 at 8 30 12 PM
Screen Shot 2022-07-10 at 8 30 24 PM

### CODE REVIEW

File: packages/js/app/src/components/TextInputBox.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import {View, StyleSheet,TextInput, Text} from 'react-native';
+import {FontAwesomeIcon} from "@fortawesome/react-native-fontawesome";
+import {faAsterisk} from "@fortawesome/free-solid-svg-icons/faAsterisk";
+import colors from '../constants/Colors';
+
+const styles = StyleSheet.create({
+    container:{
+        flexDirection: 'column',
+        padding: 10,
+    },descriptors:{
```
Comment (jbis9051) [inline packages/js/app/src/components/TextInputBox.tsx]: Styling

File: packages/js/app/src/views/LoginScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    },otherLoginButtons:{
+        height: 40,
+        width: 40,
+        margin: 7,
+        borderWidth: 1,
+        borderRadius: 20,
+        padding: 10,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },buttonText:{
```
Comment (jbis9051) [inline packages/js/app/src/views/LoginScreen.tsx]: More styling

File: packages/js/app/App.tsx [hunk 3]
```diff
@@ -1,8 +1,50 @@
...
+                    name="Login"
+                    component={login}
+                    options={{headerShown: false, title: "Log In"}}
+                />
+                <RootStack.Screen
+                    name="Signup1"
+                    component={signup1}
+                    options={{headerShown: false, title: "Sign Up"}}
+                />
+                <RootStack.Screen
```
Comment (jbis9051) [inline packages/js/app/App.tsx:38]: What's SignUp1 vs SignUp2?

File: packages/js/app/App.tsx [hunk 4]
```diff
@@ -1,8 +1,50 @@
 import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
-
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import login from './src/views/LoginScreen';
+import signup1 from './src/views/SignupScreens/Signup1';
+import signup2 from './src/views/SignupScreens/Signup2';
```
Comment (jbis9051) [inline packages/js/app/App.tsx]: Capitalize

File: packages/js/app/App.tsx (hunk 3 above)
Comment (SeaUrc) [reply packages/js/app/App.tsx:38]: I had a second sign up page which asked for more information but did not use it

Comment (johnnyr26) [reply packages/js/app/App.tsx:38]: Prob best to get rid of unused sign up page before merging with master

**Comment by jbis9051** [comment]
Please run prettier

**Comment by jbis9051** [comment]
Splash screen will need to be updated to use logo (after merge)

**Comment by jbis9051** [comment]
I don't love the use of raster graphics, can we switch to vector?

File: packages/js/app/src/views/SplashScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        }, 2000
+    );
+    return (
+        <View style={styles.container}>
+            <ImageBackground
+                source={require('../assets/background.png')}
+                style={styles.backgroundImage}
+            >
+                <View style={styles.titleContainer}>
+                    <Text style={styles.title}>Bubble</Text>
```
Comment (johnnyr26) [inline packages/js/app/src/views/SplashScreen.tsx]: You should maybe change the font color to white and add the bubble logo to the splash screen.

File: packages/js/app/src/views/SignupScreens/Signup2.tsx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        borderRadius: 15,
+        padding: 10,
+        alignItems:'center',
+    },
+})
+
+function Signup({route, navigation}: Props) {
+    return (
+        <View style={styles.container}>
+            <ImageBackground
```
Comment (johnnyr26) [inline packages/js/app/src/views/SignupScreens/Signup2.tsx]: I made a component for this so you might want to use my component after I merge mine. Try to avoid ImageBackground from now on.

File: packages/js/app/src/views/SignupScreens/Signup2.tsx [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    },titleContainer:{
+        borderWidth: 1,
+        top: '12%',
+        flex: 1.75,
+        justifyContent: 'center',
+    }, signupContainer:{
+        borderWidth: 1,
+        flex: 2.5,
+        justifyContent: 'center',
+    },signupButtonContainer:{
```
Comment (johnnyr26) [inline packages/js/app/src/views/SignupScreens/Signup2.tsx]: go to packages/js and run 'yarn prettier:fix'. Will get rid of styling issues like these

File: packages/js/app/src/views/SignupScreens/Signup2.tsx [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+        padding: 10,
+        alignItems:'center',
+    },
+})
+
+function Signup({route, navigation}: Props) {
+    return (
+        <View style={styles.container}>
+            <ImageBackground
+                source={require('../../assets/SignUp2.png')}
```
Comment (johnnyr26) [inline packages/js/app/src/views/SignupScreens/Signup2.tsx]: Does this background png look good on all screens and doesn't touch the texts below? Might want to check that.

File: packages/js/app/src/components/Header.tsx [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+import {FontAwesomeIcon} from "@fortawesome/react-native-fontawesome";
+import {faArrowLeftLong} from "@fortawesome/free-solid-svg-icons/faArrowLeftLong";
+import { useNavigation } from '@react-navigation/native';
+
+const styles = StyleSheet.create({
+    header: {
+        flexDirection: "row",
+        width: '100%',
+        height: 95,
+        backgroundColor: 'rgba(255, 255, 255, 0)',
```
Comment (johnnyr26) [inline packages/js/app/src/components/Header.tsx]: try to use hex when possible

**Review by johnnyr26 — COMMENTED** [review]
Overall, good job. Mostly styling stuff that can be done simply through yarn prettier:fix and then yarn lint:check.

File: packages/js/app/App.tsx (hunk 3 above)
Comment (SeaUrc) [reply packages/js/app/App.tsx:38]: im adding it back in because I cannot fit 
{
  "username": "<username>",
  "password": "<password>",
  "email": "<email",
  "phone": "<phone>",
  "name": "<name>"
}
required for signing up into one page

File: packages/js/app/src/views/SignupScreens/Signup2.tsx (hunk 8 above)
Comment (SeaUrc) [reply packages/js/app/src/views/SignupScreens/Signup2.tsx]: yes it looks fine although i need to adjust the input boxes

Comment (SeaUrc) [reply packages/js/app/src/views/SignupScreens/Signup2.tsx]: changing to vector

File: packages/js/app/package.json [hunk 10]
```diff
@@ -31,13 +34,15 @@
         "@babel/runtime": "^7.12.5",
         "@react-native-community/eslint-config": "^2.0.0",
         "@types/jest": "^26.0.23",
+        "@types/react": "~17.0.21",
+        "@types/react-dom": "^18.0.5",
         "@types/react-native": "^0.67.3",
         "@types/react-test-renderer": "^17.0.1",
         "@typescript-eslint/eslint-plugin": "^5.17.0",
-        "@typescript-eslint/parser": "^5.17.0",
```
Comment (jbis9051) [inline packages/js/app/package.json:37]: Why was this removed?
